R/ctwas_wrapper.R

```diff
@@ -344,37 +260,35 @@ twas_weights_loader <- R6Class("twas_weights_loader",
       return(weights = self$data$weights)
     },
     get_susie_obj = function() {
-      return(lapply(self$data$susie_results, function(context_data) {
-        context_data$susie_result[c("lbf_variable", "X_column_scale_factors", "mu")]
-      }))
+      return(lapply(self$data$susie_results, function(context_data) context_data$susie_result))
     },
     get_variant_names = function() {
```

Comment:
Again, variant names should not be part of susie_results. It should already be in the TWAS database itself.

Reply:
this is fixed - previously there were duplicates of variant names from twas results and susie_results